progetto/ServerProject/src/server/AES.java

```diff
@@ -0,0 +1,41 @@
+package server;
```

Comment:
In pratica questa classe cripta una stringa e la restituisce in base64 in modo che sia anche leggibile

Reply:
La classe non è statica in questo modo ogni thread si può creare la sua classe per crypt/decrypt